### DISCUSSION TITLE
GitHub Education - Pack

### DISCUSSION THREAD

**Discussion by OctavianAdv** [community/community#144473] [GitHub Education]
Hello i was verified in November 11 2024 , and it's near 5 days . Can you help me to get the benefits , the pack ?

**Comment by davevad93** [comment]
Hi @OctavianAdv ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @OctavianAdv was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Comment by lllHunterlllSV** [comment]
**Hello! 👋**

Thank you for reaching out with this question!

The **GitHub Student Developer Pack** is typically available to students **for as long as they are actively enrolled in an accredited educational institution**. Here are some key details:

- 📅 **Duration**: Initially, the Pack is valid for **one year**. However, students can renew it annually as long as they can verify their student status. 
- 🎓 **Eligibility**: To renew, students need to confirm their eligibility with GitHub by providing proof of enrollment (like a student ID, transcript, or other documentation).
- 🔄 **Renewal Process**: GitHub usually sends an email reminder when it's time to renew, so keep an eye on your inbox! You can also check your [GitHub Education Dashboard](https://education.github.com/discount_requests) for renewal options.
- 💡 **Important Note**: Eligibility requirements may vary based on your location or institution, so it’s always a good idea to verify current guidelines from GitHub Education.
- 🔗 **More Info**: For the latest details on eligibility and renewals, check the [GitHub Education documentation](https://education.github.com/discount_requests/pack_application).

If you have any specific questions about your account or if your status has recently changed, contacting [GitHub Support](https://support.github.com/) can also be helpful. 😊 

Hope this helps, and best of luck with all your coding adventures!
